app/assets/stylesheets/application.scss

```diff
@@ -273,6 +273,7 @@ table {
 
 .table-fixed {
   table-layout: fixed;
+  margin: 0 !important;
```

Comment:
We can do #pagination {margin-top: -6px;} ,
do you this is right? with the - sign?